### PR TITLE
bumping versions

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Proper semantic versioning plus resolving lerna issue:

```
Changes:
 - @aws-amplify/ui: 1.0.9-unstable.0 => 1.0.9-beta.0
 - aws-amplify-angular: 2.0.12-unstable.3 => 2.0.12-beta.0
 - aws-amplify-react-native: 2.0.8 => 2.0.9-beta.0
 - aws-amplify-react: 2.1.3-unstable.3 => 2.1.3-beta.0
 - aws-amplify-vue: 0.1.6-unstable.0 => 0.1.6-beta.0

lerna info auto-confirmed 
lerna ERR! execute caught error
lerna ERR! Error: fatal: tag 'aws-amplify-angular@2.0.12-beta.0' already exists
lerna ERR! 
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/aws-amplify-angular/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-amplify-angular",
-  "version": "2.0.12-unstable.3",
+  "version": "2.1.0-unstable.0",
```
Comment (powerful23) [inline packages/aws-amplify-angular/package.json]: I think you can get rid of the unstable

File: packages/aws-amplify-react/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-amplify-react",
-  "version": "2.1.3-unstable.3",
+  "version": "2.2.0-unstable.0",
```
Comment (powerful23) [inline packages/aws-amplify-react/package.json]: the same.

File: packages/aws-amplify-vue/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-amplify-vue",
-  "version": "0.1.6-unstable.0",
+  "version": "0.2.0-unstable.0",
```
Comment (powerful23) [inline packages/aws-amplify-vue/package.json]: the same.

**Comment by haverchuck** [comment]
made requested changes

**Pull request merged**